packages/rest/src/controllers/tenants/TenantsController.ts

```diff
@@ -0,0 +1,30 @@
+import type { TenantsRecord } from './TenantsControllerTypes'
+
+import { CreateTenantOptions } from '@credo-ts/tenants'
+import { Body, Controller, Post, Request, Route, Security, Tags } from 'tsoa'
+import { injectable } from 'tsyringe'
+
+import { RequestWithRootTenantAgent } from '../../authentication'
+
+import { tenantRecordToApiModel } from './TenantsControllerTypes'
+
+@Tags('Tenants')
+@Route('/tenants')
+@Security('tenants', ['admin'])
```

Comment:
what is the difference bwteen admin and default?

Reply:
Admin can only be accessed as tenant admin, which is only availalbe when the tenant module is active (--multi-tenant enabled). So it means you can't call these endpoints if tenants module is not enabled.